src/module.ts

```diff
@@ -7,11 +7,17 @@ import { MetricsPanelCtrl, loadPluginCss } from 'grafana/app/plugins/sdk';
 import * as _ from 'lodash';
 
 
+export enum TitleViewOptions {
+  AUTO = 'auto',
```

Comment:
Why is it called "AUTO"? Please explain. How this style looks like?

Reply:
Rename to `Separate title line`.

Style looks like :
![image](https://user-images.githubusercontent.com/39257464/69042175-d4396e00-0a01-11ea-88ea-b4a140d9763b.png)